Auth/class.LDAPAuthenticator.php

```diff
@@ -190,11 +203,13 @@ public function login($username, $password)
         }
         $user = $user[0];
         $server->unbind();
+
         $ret = $server->bind($user->dn, $password);
         if($ret !== false)
         {
-            return array('res'=>true, 'extended'=>$user); 
+            return array('res'=>true, 'extended'=>$user);
         }
+        $server->unbind();
```

Comment:
in practice this user will be rebound when the system starts reading for their group assignments.. the only potential conflict i see was what was previously stated..